src/App.js

```diff
@@ -15,6 +22,7 @@ function App() {
           <Route path="/login" element={<Login />} />

```

Comment:
you can move the routes into a diff routes component , to make it look more clean

Reply:
Done